shared/src/main/scala/squants/market/MoneyContext.scala

```diff
@@ -34,6 +35,26 @@ case class MoneyContext(
     allowIndirectConversions: Boolean = true) {
 
   /**
+    * Custom implementation using SortedSets to ensure consistent output
+    * @return String representation of this instance
+    */
+  override def toString: String = string
```

Comment:
I wonder if we could just map over `currencies` to get their symbols and make a `SortedSet` from that. And do the same for `rates`. That would avoid needing to create `Ordering` instances for `Currency` and `CurrencyExchangeRate`.

Something like:

```scala
val cSet = currencies.map { _.code }.sorted.mkString(',')
```